test/unit_tests/test_ops.py

```diff
@@ -125,6 +145,25 @@ def _assert_decomposition(pl_op, params=None):
     contraction_parameters.append(new_indices)
 
     actual_matrix = np.reshape(np.einsum(*contraction_parameters), [dimension, dimension])
-    pl_op_matrix = pl_op._matrix(*params) if params else pl_op._matrix()
+    pl_op_matrix = pl_op.compute_matrix(*params) if params else pl_op.compute_matrix()
 
     assert np.allclose(actual_matrix, pl_op_matrix)
+
+
+@pytest.mark.parametrize("pl_op, braket_gate", gates_2q_parametrized)
+@pytest.mark.parametrize("angle", [(i + 1) * math.pi / 12 for i in range(12)])
+def test_gate_adjoint(pl_op, braket_gate, angle):
```

Comment:
add a test for gates_2q_non_parametrized too. Not sure why coverage didn't complain.